openedx/core/djangoapps/content/learning_sequences/README.rst

```diff
@@ -0,0 +1,23 @@
+Learning Sequences
+------------------
+
+This package creates a ModuleStore-independent representation of learning
+sequences (a.k.a. "subsections" in Studio), as well as how they are put together
+into courses. It is intended to serve Course Outline and Sequence metadata
+requests to end users though the LMS, though it is also available to the Studio
+process for pushing data into the system.
+
+Direction: Keep
+===============
+
+This package is being actively developed.
+
+Usage
+=====
+
+* You may make foreign keys to learning_sequence models from your own app.
```

Comment:
Ahhh... this feels like an escape hatch to allow people to get around the restrictions inherent in the API returning `Data` classes.

Reply:
Yeah. As you pointed out, that's one of the big weaknesses with this general arrangement–avoiding n+1 queries and being able to query against it in a join. I'm hoping that this escape hatch is sufficient to get efficient querying without relying too much on the internals.